Add Tailwindcsss ,Vue and Svelte structured properly with `Learn with Sumit` resources (#6882)
EbookFoundation/free-programming-books: courses/free-courses-bn.md

* [Go](#go)
* [HTML and CSS](#html-and-css)
  * [Bootstrap](#bootstrap)

>   * [Tailwind](#Tailwind)

Reviewer: use lower case in fragment ids

Reviewer 2: Patch suggested by Eric

```suggestion
  * [Tailwind](#tailwind)
```

* [IDE and Editors](#ide-and-editors)
* [Java](#java)
* [JavaScript](#javascript)
  * [jQuery](#jquery)
  * [Node.js](#nodejs)
  * [React](#react)

>   * [Vue](#Vue)

Reviewer: use lower case in fragment ids

Reviewer 2: ```suggestion
  * [Vue](#vue)
```

* [Bootstrap 5 and 4 Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oC_wmWEZa2xWxJauIRQ9kG) - Anisul Islam

#### Tailwind

> * [Tailwind CSS Bangla Tutorial Series](https://youtube.com/playlist?list=PLHiZ4m8vCp9P23SqlHL0QAqiwS_oCofV2) - Learn with Sumit - LWS - Bangladesh

Reviewer 3: ```suggestion
* [Tailwind CSS Bangla Tutorial Series](https://youtube.com/playlist?list=PLHiZ4m8vCp9P23SqlHL0QAqiwS_oCofV2) - Sumit Saha
```

* [React JS Tutorial Bangla Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9M6HVQv7a36cp8LKzyHIePr) - Sumit Saha (Learn with Sumit)

#### Vue

* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/watch?v=iIvN7upsLoA&list=PLHiZ4m8vCp9O5_QoMvQbOAOR5WolyZ4OS&index=7&t=1989s) - Learn with Sumit

#### Svelte

> * [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner

Reviewer 3: ```suggestion
* [Explore Programming in Bangla ( বাংলা )](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9MrJGkaQGLJHapIz5goTqgl) - Learn with Sumit
* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner
```

* [React JS Tutorial Bangla Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9M6HVQv7a36cp8LKzyHIePr) - Sumit Saha (Learn with Sumit)

#### Vue

> * [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://youtu.be/iIvN7upsLoA) - Learn with Sumit

Reviewer: We do NOT accept shortened links.

> * [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://youtu.be/iIvN7upsLoA) - Learn with Sumit - LWS - Bangladesh

Reviewer 3: ```suggestion
* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/watch?v=iIvN7upsLoA) - Sumit Saha
```
Shortened URLs are not allowed

#### Svelte

* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner

> * [Svelte Bangla Overview Tutorial with examples - How Svelte is different from React & others](https://youtu.be/DdFcbIsqPh8) - Learn with Sumit

Reviewer: We do NOT accept shortened links.

> * [Svelte Bangla Overview Tutorial with examples - How Svelte is different from React & others](https://www.youtube.com/watch?v=DdFcbIsqPh8&list=PLHiZ4m8vCp9MrJGkaQGLJHapIz5goTqgl&index=11&t=820s) - Learn with Sumit

Author: ???

* [React JS Tutorial Bangla Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9M6HVQv7a36cp8LKzyHIePr) - Sumit Saha (Learn with Sumit)

#### Svelte

* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner

#### Vue

> * [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/playlist?&list=PLHiZ4m8vCp9O5_QoMvQbOAOR5WolyZ4OS) - Learn with Sumit

Reviewer 2: Revert to video url, which is 1h length enough

```suggestion
* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/watch?v=iIvN7upsLoA) - Learn with Sumit
```
